woocommerce/woocommerce-android
# Issue/4465 Crash on tutorial during the payment flow

Resolves #4465 

The PR fixes:
* Dark mode background color for the tutorial
* Crash during payments flow

~~Currently, there is a bug that after the tutorial we don't show payment collection dialog~~
workaround - https://github.com/woocommerce/woocommerce-android/pull/4474/commits/5ee09388c54a3a005c34e94949fcf01e5a7799e2

### How to test
1.  Clear apps data to make sure the "tutorial seen" flag gets cleared
2. Star the app - make sure the store is eligible for in person payments
3. Open detail of an unpaid order in USD
4. Tap on "Collect payment" button
5. Notice the app starts connecting to the reader and shows the tutorial after

---

1.  Clear apps data to make sure the "tutorial seen" flag gets cleared
2. Star the app - make sure the store is eligible for in person payments
3. Open settings
4. Open "manager reader"
5. Connect to a reader
6. Notice the app starts connecting to the reader and shows the tutorial after

The bug is critical, so it has to be fixed before the cut-off, but I'm afk tomorrow, so @NickGreen or @malinajirka please go ahead and make changes in the code if necessary so we can merge it tomorrow.
